pyQuARC/code/custom_validator.py

```diff
@@ -167,3 +171,22 @@ def characteristic_name_uniqueness_check(characteristics):
             "valid": not bool(duplicates),
             "value": ', '.join(duplicates)
         }
+
+    @staticmethod
+    def get_data_url_check(metadata_json):
+        required_type = 'GET DATA'
+        related_urls = metadata_json.get('Related_URL')
```

Comment:
`related_urls = metadata_json.get('Related_URL', [])`

then remove the if condition at 181.

Reply:
done d052528